sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/api/Azure.Monitor.OpenTelemetry.LiveMetrics.net6.0.cs

```diff
@@ -0,0 +1,106 @@
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics
+{
+    public partial class LiveMetricsExporterOptions : Azure.Core.ClientOptions
+    {
+        public LiveMetricsExporterOptions() { }
+        public string ConnectionString { get { throw null; } set { } }
+        public Azure.Core.TokenCredential Credential { get { throw null; } set { } }
+        public bool EnableLiveMetrics { get { throw null; } set { } }
+    }
+    public static partial class LiveMetricsExtensions
+    {
+        public static OpenTelemetry.Logs.OpenTelemetryLoggerOptions AddLiveMetrics(this OpenTelemetry.Logs.OpenTelemetryLoggerOptions loggerOptions, System.Action<Azure.Monitor.OpenTelemetry.LiveMetrics.LiveMetricsExporterOptions> configure = null) { throw null; }
+        public static OpenTelemetry.Trace.TracerProviderBuilder AddLiveMetrics(this OpenTelemetry.Trace.TracerProviderBuilder builder, System.Action<Azure.Monitor.OpenTelemetry.LiveMetrics.LiveMetricsExporterOptions> configure = null, string name = null) { throw null; }
+    }
+}
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Models
+{
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct DerivedMetricInfoAggregation : System.IEquatable<Azure.Monitor.OpenTelemetry.LiveMetrics.Models.DerivedMetricInfoAggregation>
```

Comment:
Why is this a public API?

Reply:
This will go away.

Builds are currently failing because our api files are out of sync.
I'm addressing this in https://github.com/Azure/azure-sdk-for-net/pull/42919